sentences/en/light_HassLightSet.yaml

```diff
@@ -16,20 +21,31 @@ intents:
           - "(set | change) <area> to <brightness>"
         slots:
           name: "all"
+        response: "brightness_area"
+      - sentences:
+          - "(set | change) <area> to <brightness>"
+        requires_context:
+          domain: "light"
```

Comment:
So, how is this supposed to be working? What does it control/change in the logic? Does it just means that this will set all (set by `slots.name: "all"`) `light` domain devices to `<brightness>`?

Reply:
This means that as the intent matcher is running, if these sentences match the user's text then the domain of the target entity/entities must **also** be light. Otherwise, the match is skipped and the process continues.

It was intended to cases where the same sentence would trigger two intents, so extra info was needed to disambiguate.